src/lib/editableGrid/Cell.tsx

```diff
@@ -90,6 +91,29 @@ export default function Cell({
     }
   }
 
+  /**
+   * This function is needed to select text in number-cells via first click on the cell
+   */
+  function onClick(e: React.MouseEvent<Element>) {
+    if (e.target instanceof HTMLDivElement && e.detail === 1) {
```

Comment:
what if `<div><span>smth</span></div>` and user's click happened on <span>? - need cover by test